setup.cfg

```diff
@@ -1,30 +0,0 @@
-[tool:pytest]
-addopts =
-    --import-mode=importlib
-    --doctest-modules
-    --ignore=archive
-    --ignore=doc/sphinx/source/conf.py
-    --cov=esmvaltool
-    --cov-report=xml:test-reports/coverage.xml
-    --cov-report=html:test-reports/coverage_html
-    --html=test-reports/report.html
-    --numprocesses auto
-env =
-    MPLBACKEND = Agg
-log_level = WARNING
-markers =
-    installation: test requires installation of dependencies
-    diagnostic_image_output: test produces images that will be compared with imagehash
-
-[coverage:run]
-parallel = true
-
-[codespell]
-ignore-words-list = vas,runn
-
-[pycodestyle]
-# ignore rules that conflict with ruff formatter
-# E203: https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#slices
-# E501: https://docs.astral.sh/ruff/formatter/#conflicting-lint-rules
-# W503: https://pycodestyle.pycqa.org/en/latest/intro.html#error-codes
-ignore = E203,E501,W503
```

Comment:
This needs to stay: pycodestyle does not support pyproject.toml

Reply:
pycodestyle is called by prospector, so it needs to stay until we have moved over to ruff completely

Reply:
good point! Just realized it's in Core too, am putting it back

Reply:
done in 6e7ba909617ae0730071738fc0ab79a4dd03e7a2